docs/manual-install.md

```diff
@@ -137,7 +137,7 @@ This guide demonstrates steps required to enable the KMS Plugin for Key Vault in
 
 ### 4. Create encryption configuration
 
-  Create a new encryption configuration file `/etc/kubernetes/manifests/encryptionconfig.yaml` using the appropriate properties for the `kms` provider:
+  Create a new encryption configuration file `/etc/kubernetes/enc/encryptionconfig.yaml` using the appropriate properties for the `kms` provider:
```

Comment:
These bits look important to get right, I imagine incorrect paths will result in a broken install.  I'm not familiar off the top of my head.

@enj @aramase for 👀 ?